xml-html-conduit-lens.cabal

```diff
@@ -68,7 +68,7 @@ test-suite hspec
     , hspec
     , hspec-expectations-lens >= 0.3
     , lens                    >= 4.0
-    , xml-conduit             >= 1.1   && < 1.5
+    , xml-conduit             >= 1.1   && < 1.6
```

Comment:
Shouldn't this upper bound be `< 1.8` as well?

Reply:
Good catch, thanks! I fixed it :)